src/qt/res/qml/WalletPane.qml

```diff
@@ -0,0 +1,201 @@
+            height: parent.height / 3 * 2
+            contentItem: transactionInfoFlickable
+
+            Flickable {
+                id: transactionInfoFlickable
+
+                contentWidth: transactionInfoText.width
+                contentHeight: transactionInfoText.height
+
+                clip: true
+
+                ScrollBar.vertical: ScrollBar {
+                    parent: transactionInfoFlickable.parent
+                    anchors.top: transactionInfoFlickable.top
+                    anchors.left: transactionInfoFlickable.right
+                    anchors.bottom: transactionInfoFlickable.bottom
+                }
+
+                Text {
+                    id: transactionInfoText
+                    width: transactionInfoDialog.availableWidth
+                    text: transactionInfoDialog.transactionInfoString.slice(0, -18) // Get rid of the last newline
+                    wrapMode: Text.Wrap
+                    font: theme.thinFont
+                    color: primaryColor
+                }
+            }
+        }
+
+        ToolBar {
+            Material.elevation: 0
+            Material.foreground: primaryColor // TODO: investigate why these are not propagated from main
+            Layout.fillWidth: true
+            RowLayout {
+                anchors.fill: parent
+                ToolButton {
+                    Layout.alignment: Qt.AlignHCenter
+                    text: bitcoinTr("BitcoinGUI","&Send")
+                    onClicked: {
+                        sendPane.camera.start()
+                        stackView.push(sendPane)
+                    }
+                    font: theme.thinFont
+                }
+                ToolButton {
+                    text: bitcoinTr("BitcoinGUI","&Receive")
+                    Layout.alignment: Qt.AlignHCenter
+                    onClicked: request()
+                    font: theme.thinFont
+                }
+
+                ToolButton {
+                    text: "⋮"
+                    Layout.alignment: Qt.AlignRight
+
+                    onClicked: optionsMenu.open()
+
+                    Menu {
+                        Material.foreground: primaryColor
+                        Material.background: secondaryColor
+
+                        id: optionsMenu
+                        x: parent.width - width
+                        transformOrigin: Menu.TopRight
+
+                        MenuItem {
+                            text: bitcoinTr("BitcoinGUI", "&Settings")
+                            font: theme.thinFont
+                            onTriggered: {
+                                stackView.push(settingsPane)
+                            }
+                        }
+
+                        MenuItem {
+                            text: qsTr("About")
+                            font: theme.thinFont
+                            onTriggered: {
+                                stackView.push(aboutPane)
```

Comment:
I think this is in the wrong place. This is similar to use global variables and IMO it's fragile (but convenient). I'd suggest to add a property `required Action aboutAction` to this component and set it in main.qml.